modules/openapi-generator/src/main/resources/typescript-inversify/api.service.mustache

```diff
@@ -52,7 +54,7 @@ export class {{classname}} {
      * {{summary}}
      * {{notes}}
      {{#allParams}}* @param {{paramName}} {{description}}
-     {{/allParams}}{{#useHttpClient}}* @param observe set whether or not to return the data Observable as the body, response or events. defaults to returning the body.
+     {{/allParams}}* {{#useHttpClient}}@param observe set whether or not to return the data as the body, response or events. defaults to returning the body.
```

Comment:
Please revert this change. Moreover, it is best to create separate PRs for unrelated changes to make the review a bit easier